.ci/docker/common/install_cuda.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-NCCL_VERSION=v2.25.1-1
+NCCL_VERSION=v2.26.2-1
```

Comment:
What about arm arch?

Reply:
@Skylion007 aarch64 uses same script: https://github.com/pytorch/pytorch/actions/runs/14011332351/job/39231501156#step:6:1018
So we should be good

Reply:
Oh, is this script redundant then? https://github.com/pytorch/pytorch/blob/db9b031b0097d68c14f32d9c58609b8b89930426/.ci/docker/common/install_cuda_aarch64.sh

Reply:
Yes, you are correct. This is still used. We will be consolidating these scripts: https://github.com/pytorch/pytorch/issues/149554